pkg/config/value_constraints.go

```diff
@@ -0,0 +1,78 @@
+package config
+
+type DistributionVersion struct {
+	Name  string
+	Label string
+}
+type DistributionArch struct {
+	Name  string
+	Label string
+}
+
+const El7 = "el7"
+const El8 = "el8"
+const El9 = "el9"
+
+var DistributionVersions = [...]DistributionVersion{
+	{
+		Name:  "el7",
+		Label: El7,
+	}, {
+		Name:  "el8",
+		Label: El8,
+	}, {
+		Name:  "el9",
+		Label: El9,
+	},
+}
+
+const X8664 = "x86_64"
+const S390x = "s390x"
+const PPC64LE = "ppc64le"
+const AARCH64 = "aarch64"
+
+var DistributionArches = [...]DistributionArch{
+	{
+		Name:  "x86_64",
+		Label: X8664,
+	}, {
+		Name:  "s390x",
+		Label: S390x,
+	}, {
+		Name:  "ppc64le",
+		Label: PPC64LE,
+	}, {
+		Name:  "aarch64",
+		Label: AARCH64,
+	},
+}
+
+// ValidDistributionVersionLabels Given a list of labels, return true or false
```

Comment:
```suggestion
// ValidDistributionVersionLabels Given a list of labels, return true
```

Reply:
its actually part of the next statement  "return true if every item of the list is a valid distribution version."

Reply:
yeah but right now it says "return true or false if every item of the list is a valid distribution version."

Reply:
oh derp derp, will fix :0